lib/vcr/test_frameworks/rspec.rb

```diff
@@ -8,7 +8,12 @@ module Metadata
       def configure!
         ::RSpec.configure do |config|
           vcr_cassette_name_for = lambda do |metadata|
-            description = metadata[:description]
+            description = if metadata[:description].empty?
+                            # we have an "it { is_expected.to be something }" block
+                            metadata[:scoped_id]
+                          else
+                            metadata[:description]
+                          end
```

Comment:
Are we sure the state of things is that `metadata[:description]` will either be a full string `"something"` or an empty string `""`?